ansible_ai_connect/main/settings/base.py

```diff
@@ -612,3 +612,9 @@ def is_ssl_enabled(value: str) -> bool:
 CHATBOT_DEFAULT_PROVIDER = os.getenv("CHATBOT_DEFAULT_PROVIDER")
 CHATBOT_DEFAULT_MODEL = os.getenv("CHATBOT_DEFAULT_MODEL")
 # ==========================================
+
+# ==========================================
+# Playbook Generation/Explanation endpoints
+# ------------------------------------------
+ENABLE_PLAYBOOK_ENDPOINT = os.getenv("ENABLE_PLAYBOOK_ENDPOINT", "False").lower() == "true"
```

Comment:
I would prefer to prefix these envvars with `ANSIBLE_AI` (or some other prefix to discuss) to simplify the identification of the different keys, e.g: `ANSIBLE_AI_ENABLE_PLAYBOOK_ENDPOINT`.

Can you turn if on by default? I believe this is how we expect it be be initialized.

Reply:
@goneri I'll change the name; no problem.

Changing the default behaviour to "enabled" is however more problematic.

An older version of the Ansible AI Connect Operator would not be aware of the new setting and if a User changes the definition of their "on prem" Ansible AI Connect instance to reference the latest service image they'd get Playbook Generation and Explanation "for free" (as it would default to enabled).

Keeping the default as disabled ensures we don't leak features for "on prem".

I did a [PR](https://github.com/ansible/ansible-wisdom-ops/pull/1096) for `ansible-wisdom-ops` to ensure Playbook Generation and Explanation are enabled for our SaaS offering (I will need to update with the name change) and the _development_ settings enable it too.

Do you see an alternative?

Reply:
I understand, what about something like:

```
ENABLE_PLAYBOOK_ENDPOINT = os.getenv("ENABLE_PLAYBOOK_ENDPOINT", "False" if settings.DEPLOYMENT_MODE == "on-prem" else "True")).lower() == "true"
```